allauth/templates/account/email.html

```diff
@@ -44,14 +44,19 @@ <h1>{% trans "E-mail Addresses" %}</h1>
 
 {% endif %}
 
-
+  {% if can_add_emails %}
     <h2>{% trans "Add E-mail Address" %}</h2>
 
+    {% if account_email_limit and account_email_limit > 1 %}
```

Comment:
From a less-is-more perspective, I think it is more user friendly to not constantly present this limit to the user. Likely, under normal circumstances, users should not be hitting the limit anyway... so I would drop this.